package.json

```diff
@@ -113,6 +113,7 @@
     "babel-loader": "9.2.1",
     "babel-plugin-transform-class-properties": "6.24.1",
     "concurrently": "9.1.0",
+    "cors": "^2.8.5",
```

Comment:
Please remove the ^ symbol, we pin all dependency versions.

Reply:
done